exercises/performance/jmh-benchmarking/pom.xml

```diff
@@ -60,12 +60,12 @@ THE POSSIBILITY OF SUCH DAMAGE.
         <!--
```

Comment:
Do we need a POM, which I assume means using Maven? The testing exercise repo had JMH using a Gradle plugin: https://github.com/sweng-epfl/testing-lecture-exercises/blob/main/build.gradle

Reply:
I'll change to use gradle